examples/hubert/train.py

```diff
@@ -90,7 +92,7 @@ def _parse_args():
     )
     parser.add_argument(
         "--resume-checkpoint",
-        type=Optional[pathlib.Path],
+        type=pathlib.Path,
```

Comment:
This should remain `Optional[pathlib.Path]`

Reply:
actually using `Optional` in argparse will throw an error, that's why I removed those from the parser.